docs/1.0.0/install_macos.md

```diff
@@ -0,0 +1,230 @@
+2. An RSK account with an RBTC balance NOT lower than 0.001 RBTC
+3. XCode
+
+## Install required libraries/software
+
+### Install command line tools
+
+```
+xcode-select --install
+```
+
+### Install Homebrew
+
+From a Terminal window run:
+
+```
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+### Install OpenSSL
+
+```
+brew install openssl
+export LC_ALL="en_US.UTF-8"\nexport LC_CTYPE="en_US.UTF-8"
+export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
+```
+
+### Install Python 3.7
+
+
+```
+brew install python3
+```
+
+### Install PIP
+
+```
+sudo easy_install pip
+```
+
+### Install virtualenv
+
+
+
+```
+pip3 install virtualenv
+```
+
+### Install libpq
+
+```
+brew install libpq
+```
+
+### Install postgresql
+
+```
+brew install postgresql
+```
+
+### Install psycopg2
+
+```
+env LDFLAGS="-I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib" pip install psycopg2
+```
+
+
+## Build RIF Lumino from code
+
+1. Get the code from [https://github.com/rsksmart/lumino/releases/tag/1.0.0](https://github.com/rsksmart/lumino/releases/tag/1.0.0)
+2. Uncompress the downloaded file
+2. Go to the path you uncompressed the code in the previous step (lets call this path `$RIF_LUMINO_PATH`)
+3. Create python virtual env for RIF Lumino (this needs to be performed only once) and execute the following command:
+
+```
+virtualenv -p <PATH_TO_PYTHON3.7> clientEnv
+```
+
+**Note:**
+Replace `<PATH_TO_PYTHON3.7>` with the path where Python3.7 is installed in your system. In the case of MacOS, this is usually `/usr/local/bin/python3.7`
+
+4. Activate python virtual env, by executing the following command:
+
+```
+source clientEnv/bin/activate
+```
+
+5. Check if the Python version is correct inside the virtual environment by running:
+
+```
+python --version
+```
+
+This command should output version 3.7.x
+
+6. Install RIF Lumino requirements. Inside the virtual environment run the following command (this could take a few minutes):
+
+```
+pip install -c constraints.txt --upgrade -r requirements-dev.txt
+```
+
+7. Run Lumino setup with the following command:
+
+```
+python setup.py develop
+```
+
+## Start RIF Communications transport layer
+
+The comunication between Lumino nodes can be done both using RIF Communications (https://www.rifos.org/communications) and Matrix (https://matrix.org/).
+
+The default way, and the one we encourage to use for a more decentralized ecosystem, is RIF Communications. 
+
+In order to run Lumino using RIF Comms, you need to set up the RIF Communications bootnode. To use Matrix, no configuration is required. 
+
+### Set up a RIF Communications bootnode
+
+#### Requirements:
+
+- Node 14+ (14.13 recommended)
+
+#### Get the code
+
+- git clone https://github.com/rsksmart/rif-communications-pubsub-bootnode/tree/grpc-api 
```

Comment:
Modify this to 

- get clone
- checkout grpc api branch

Reply:
i've made a suggestion for this in the ubuntu guide